Project.toml

```
@@ -14,7 +14,7 @@ SymDict = "2da68c74-98d7-5633-99d6-8493888d7b1e"
 [compat]
 AWSCore = "0.5, 0.6"
 HTTP = "0.6, 0.7, 0.8"
-MbedTLS = "0.6, 0.7"
+MbedTLS = "0.6, 0.7, 0.8, 0.9, 1"
```

```suggestion
MbedTLS = "0.6, 0.7, 1"
```
It doesn't look like `0.8` and `0.9` tags exist https://github.com/JuliaLang/MbedTLS.jl/releases